docs/OV_Runtime_UG/auto_device_selection.md

```diff
@@ -13,30 +13,27 @@
-+----------+-------------------------------------------------+-------------------------------------+
-| 3        | | iGPU                                          | FP32, FP16, BIN,                    |
-|          | | (e.g. Intel® UHD Graphics 620 (iGPU))         |                                     |
-+----------+-------------------------------------------------+-------------------------------------+
-| 4        | | Intel® Neural Compute Stick 2 (Intel® NCS2)   | FP16                                |
-|          |                                                 |                                     |
-+----------+-------------------------------------------------+-------------------------------------+
-| 5        | | Intel® CPU                                    | FP32, FP16, INT8, BIN               |
-|          | | (e.g. Intel® Core™ i7-1165G7)                 |                                     |
-+----------+-------------------------------------------------+-------------------------------------+
++----------+------------------------------------------------------+-------------------------------------+
+| Choice   | | Supported                                          | | Supported                         |
+| Priority | | Device                                             | | model precision                   |
++==========+======================================================+=====================================+
+| 1        | | dGPU                                               | FP32, FP16, INT8, BIN               |
+|          | | (e.g. Intel® Iris® Xe MAX)                         |                                     |
++----------+------------------------------------------------------+-------------------------------------+
+| 2        | | iGPU                                               | FP32, FP16, BIN,                    |
+|          | | (e.g. Intel® UHD Graphics 620 (iGPU))              |                                     |
++----------+------------------------------------------------------+-------------------------------------+
+| 3        | | Intel® Movidius™ Myriad™ X VPU                     | FP16                                |
+|          | | (e.g. Intel® Neural Compute Stick 2 (Intel® NCS2)) |                                     |
++----------+------------------------------------------------------+-------------------------------------+
+| 4        | | Intel® CPU                                         | FP32, FP16, INT8, BIN               |
+|          | | (e.g. Intel® Core™ i7-1165G7)                      |                                     |
++----------+------------------------------------------------------+-------------------------------------+
 @endsphinxdirective
 
 To put it simply, when loading the network to the first device on the list fails, AUTO will try to load it to the next device in line, until one of them succeeds. For example: 
```

Comment:
can we provide a picture from the blob about auto plugin? it's hard to read the text w/o any illustration